android/app/src/main/java/org/fossasia/openevent/activities/MainActivity.java

```diff
@@ -486,27 +487,30 @@ public void run() {
     }
 
     private void doMenuAction(int menuItemId) {
-        final FragmentManager fragmentManager = getSupportFragmentManager();
+        //final FragmentManager fragmentManager = getSupportFragmentManager();
         addShadowToAppBar(true);
         switch (menuItemId) {
             case R.id.nav_tracks:
                 atHome = true;
-                fragmentManager.beginTransaction()
+                /*fragmentManager.beginTransaction()
                         .replace(R.id.content_frame, new TracksFragment(), FRAGMENT_TAG_TRACKS).commit();
                 if (getSupportActionBar() != null) {
                     getSupportActionBar().setTitle(R.string.menu_tracks);
                 }
-                appBarLayout.setExpanded(true, true);
+                appBarLayout.setExpanded(true, true);*/
+                replaceFragment(new TracksFragment(),getString(R.string.menu_tracks));
                 break;
             case R.id.nav_schedule:
                 atHome = false;
-                fragmentManager.beginTransaction()
+                replaceFragment(new ScheduleFragment(),getString(R.string.menu_schedule));
+
+                /*fragmentManager.beginTransaction()
```

Comment:
You'd probably like to remove a code block, not just comment it out. You can always revert a commit but committing old redundant code blocks pollutes the code and needs someone else's attention to remove them later